### PR TITLE
Edit first slides of "Creating workshop website" to warm tone

### PR DESCRIPTION
**Pull request by willingc**
Here are some edits up to the slide about Jekyll. Will push more changes as time permits.

### CODE REVIEW

File: creating-website/index.html [hunk 1]
```diff
@@ -21,45 +21,68 @@
...
-	    GitHub publishes it as a website at <code>http://user.github.io/repo/</code>.
-	  </li>
-	  <li>
-	    Plain pages published as-is.
-	  </li>
-	  <li>
-	    Pages with special headers are compiled.
-	  </li>
-	</ol>
+        <h2>How?</h2>
```
Comment (willingc) [inline creating-website/index.html:59]: Ugh...forgot to delete lines 59-66. Will push a correction.

Comment (willingc) [inline creating-website/index.html:59]: Corrected.

**Comment by gvwilson** [comment]
@PBarmby can you please review?

**Comment by PBarmby** [comment]
Looks good! Only one phrase scanned oddly to me: "this learning". Maybe "this material" or "what they've learned" ?

**Comment by willingc** [comment]
@PBarmby Thanks. I agree on your point. Do you have a preference of the two suggestions? I'm happy to push an edit to the PR. Also, I am hoping to get to the second half in the next few days.

**Comment by PBarmby** [comment]
I think "what they've learned" is my preference as it's a little less formal. 
This is the first time I've had something to review; is the procedure that I merge the PR, or do we leave that to @gvwilson ?

**Comment by gvwilson** [comment]
If you're both happy, please go ahead and merge with my thanks.

**Pull request merged**